graphics/inkscape-textext/Portfile

```diff
@@ -20,11 +20,11 @@ description             LaTeX typesetting in Inkscape
 long_description        Textext provides re-editable LaTeX objects to Inkscape’s \
                         repertoire.
 
-checksums               sha256  d4a64439c60613150a47579646a764a20c3c30be9301517305fe8ca4f2e2c121 \
-                        rmd160  91b471f57c370836bf0edaea79c51ac36812f906 \
-                        size    3556084
+checksums           sha256  bcafc662c9e65cc49d3a7aad1d25ab94f8650cd9d412442efeb3a723b2d30b44 \
+                    rmd160  bf8f0180a263b1c0459ed02bac6ea7c155f8b01d \
+                    size    3556147
 
-set python_minor        10
+set python_minor        11
```

Comment:
This doesn't work with 3.12?

Reply:
Maybe it does (didn't try), but I see that Inkscape's python runtime dependencies are py311, so i thought I would just match that.

Reply:
Would be good to update Inkscape at some point soon too. The new preferred python is 3.12.